_persons/thanhson.md

```diff
@@ -0,0 +1,28 @@
+---
+# use person if you want to have a personal website created
```

Comment:
Why are you adding yourself again?
We already have you under `nguyen.md`.
So please remove this file.

Reply:
The same goes for `thanhson.jpg`.